### ISSUE TITLE
Link to changelogs when available

### ISSUE THREAD

**Issue by vemv** [liquidz/antq#84]
Hi again,

I liked this feature:

```
Available diffs:
- https://github.com/clojure/clojure/compare/clojure-1.10.1...1.10.3
```

Additionally, one could try printing https://github.com/clojure/clojure/blob/clojure-1.10.3/changes.md if that file exists (as of that tag). A few filenames could be atttempted to be fetched:

* `changes.md`
* `CHANGELOG.md`
* `CHANGELOG.adoc`

and so on. I checked that one can query those files via HEAD so as not to spam much github's servers.

WDYT? I could give it a shot.

**Comment by liquidz** [comment]
@vemv Thanks for your suggestion!

Oh, it is the feature I've been wanteded for a long time! (I didn't have time to start yet)

> I checked that one can query those files via HEAD so as not to spam much github's servers.

Great!

**Comment by vemv** [comment]
Nice! I'll try to contribute it sometime this week

**Comment by vemv** [comment]
FYI I'm progressing well with this feature :) hope to open a PR in a few days

**Comment by liquidz** [comment]
NOTE: clj-http has `changelog.org`
https://github.com/dakrone/clj-http/blob/3.x/changelog.org

**Comment by vemv** [comment]
Hey there!

I hadn't forgotten about this one, but was hesitant about which http library to use.

I successfully pushed for https://github.com/clj-commons/clj-http-lite to move to clj-commons and to solve some outstanding tech issues.

It seems a very nice library since it's dependency-free, and it works with all JDKs.

Feel free to consider using it.

Regarding my branch and the more recent commits from your side, I wouldn't mind much discarding my work. When your branch is ready feel free to @ me, I might contribute some ideas from what I had committed 👀
